burger/src/components/Contacts/Contacts.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import mail from "../../assets/mail.png"
+import clock from "../../assets/clock.png"
+import facebook from "../../assets/facebook.png"
+import instagram from "../../assets/instagram.png"
+import phone from "../../assets/iphone.png"
+
+
+export default function Contacts() {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    axios('https://burger-api-xcwp.onrender.com/contact').then((r) =>
+      setData(r.data)
+    );
+  }, []);
+
+  return (
+    <ul>
+      {data.map((e) => (
+        <li key={e._id}>
+          <div><h3> <img src={phone} className="ingredients" alt="Logo" />{e.phone}</h3>
```

Comment:
It should be aligned better

<img width="525" alt="Screenshot 2023-01-09 at 19 35 39" src="https://user-images.githubusercontent.com/40695369/211371341-8a91be46-8ffe-4d4c-902e-dea44b0462c9.png">